src/mock/parsec/state.rs

```diff
@@ -26,15 +26,18 @@ use unwrap::unwrap;
 
 pub(super) struct SectionState<T: NetworkEvent, P: PublicId> {
     observations: BTreeMap<ObservationHolder<T, P>, ObservationState<P>>,
-    unconsensused_observations: BTreeSet<ObservationHolder<T, P>>,
+    // Unconsensused Observations in the order they were firs voted for.
```

Comment:
`in the order they were firs voted for` => `in the order they were voted for` ?

Reply:
Will fix the typo, but it is in the order they were 'first' voted for, i.e Genesis will 'first' be voted for by the 1st new parsec, and then an opaque is voted, and then Genesis will again be voted when second Parsec instance is created for the same genesis group.

Reply:
Ah, ok. thx for the clarification.